extern/game_support/stm32f4/README.txt

```diff
@@ -0,0 +1,5 @@
+This example is used by the tutorial located in the GNAT Pro User�s Guide Supplement for Cross Platforms.  The tutorial is in the Annex titled "Tutorial: Embedded ARM Ada Project." 
```

Comment:
Very small nit: Make this an MD file?

Reply:
Yes. Also I can see some weird encoding issue (ISO-8859-1?) I'll have a look.